images/runtime/dotnetcore/template.Dockerfile

```diff
@@ -41,6 +41,6 @@ RUN echo $USER_DOTNET_AI_VERSION && ln -s /opt/startupcmdgen/startupcmdgen /usr/
     && apt-get install unzip -y \ 
     && apt-get upgrade --assume-yes \
     && mkdir -p /DotNetCoreAgent \
-    && curl -o /DotNetCoreAgent/appinsights.zip "https://oryxsdksdev.blob.core.windows.net/appinsights-agent/DotNetCoreAgent.$USER_DOTNET_AI_VERSION.zip" \
+    && curl -o /DotNetCoreAgent/appinsights.zip "https://oryxsdksstaging.blob.core.windows.net/appinsights-agent/DotNetCoreAgent.$USER_DOTNET_AI_VERSION.zip" \
```

Comment:
Is this `oryxsdksstaging` value one that we can pass in as an `ARG` to this Dockerfile? That way we can set the value with `--build-arg` when we call `docker build` on this file and minimize the blast radius if we ever needed to change the name of the storage account in the future (or which storage account we wanted to use here if we needed to make it more dynamic)

Reply:
I see this `oryxsdksstaging` in a few other Dockerfiles in this PR, so this comment would apply to those files as well.

Reply:
Have you been able to run the validation and nightly tests on this and verify that this exists in `oryxsdksstaging`? Also yeah, agree w/ cormac about parametrizing